### DISCUSSION TITLE
Can I graph stats pulled in via CSV?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#142] [Q&A]
Hello,

I have a CSV being pulled in that has 'Strings' and 'Numbers', and it looks great.

I've been asked to graph the stats which are 'Number', but I just can't work it out or see if it's even possible, can some one clarify?

If so how?

**Comment by marcusolsson** [comment]
Do you know in advance which types the columns have? If you create a schema for the number fields and exclude the rest, would that work? You can also use regexps to set the type for all matched columns.

**Reply by G0nz0uk** [reply]
Hi,  sorry Marcus, I’m not sure I understand what you mean by types of columns and schema, could you explain that?

**Comment by maheswarGR** [comment]
Hi  @marcusolsson .. I have loaded csv file in grafana ..but i couldnt see graph in dashboard ..KIndly help
![Screenshot (450)](https://user-images.githubusercontent.com/91525476/146924144-08f9c22f-c87a-4256-a86e-0c88c1e077a2.png)
![Screenshot (449)](https://user-images.githubusercontent.com/91525476/146924217-1e014b2f-a356-418a-9129-eeacf7f1562b.png)
